benchmarks/microbenchmarks/test/benchmark_config.yml

```diff
@@ -0,0 +1,18 @@
+# Sample configuration for inference kernel benchmarks
+quantization_config_recipe_names:
+  - "baseline"
+  - "int8wo"
+  - "int4wo-128"
+output_dir: "benchmarks/microbenchmarks/test/results"  # Directory for results and plots
+model_params:
+  matrix_shapes:
+    - name: "custom"
+      shapes: [
+        [1024, 1024, 1024],  # [m, k, n]
+        [2048, 4096, 1024],
+        [4096, 4096, 1024]
+      ]
+  high_precision_dtype: "torch.bfloat16"
+  compile: "max-autotune"
```

Comment:
this is confusing, it is not clear what "compile" means here and this variable name does not match argument names from the `torch.compile` API.  IMO something like `use_torch_compile` and `torch_compile_mode='max_autotune'` is clearer.

Reply:
two arguments sounds right to me, IMO the number of "new arguments to remember" should be minimized, and anything that matches the same args from other functions should match the arg names from those functions exactly.  It's more LOC, but it's easier to understand with less ambiguity.